bundle/src/assembly/resources/etc/runtime.json

```diff
@@ -220,7 +220,9 @@
                 "CellularHandoverUpdates",
                 "VehicleUpdates",
                 "VehicleRoutesInitialization",
-                "VehicleTypesInitialization"
+                "VehicleTypesInitialization",
+                "ServerRegistration",
+                "TmcRegistration"
```

Comment:
Move these lines to the other "registration" interactions in this list for a better order